ospf6d/ospf6_lsdb.c

```diff
@@ -92,6 +106,52 @@ static void _lsdb_count_assert(struct ospf6_lsdb *lsdb)
 #define ospf6_lsdb_count_assert(t) ((void) 0)
 #endif /*DEBUG*/
 
+#define OSPF6_LSDB_STATS_NUM_INC 1
```

Comment:
Is this used somewhere?

Reply:
Yes. I could have used `1` and `0` constant integers in the function calls, but i thought it would make the code more readable this way. The precompiler will resolve them to those values and gcc/clang should optimize them to constants in the inline function in the same way.